### PR TITLE
some ui org tests refactored

### PR DESCRIPTION
**Pull request by pondrejk**
Grouped some add and remove entity tests to the end-to-end test so that they reuse the org and session.
It relies on  https://github.com/SatelliteQE/airgun/pull/383 to function (on some webdriver versions I guess)
Test result:

```pytest tests/foreman/ui/test_organization.py -k end_to_end
2019-07-31 13:00:08 - conftest - DEBUG - Registering custom pytest_configure

================================================================= test session starts ==================================================================
platform linux -- Python 3.7.3, pytest-4.6.3, py-1.7.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/pondrejk/Documents/robottelo
plugins: cov-2.7.1, mock-1.10.4, forked-0.2, services-1.3.1, xdist-1.29.0
collecting ... 2019-07-31 13:00:08 - conftest - DEBUG - BZ deselect is disabled in settings

collected 6 items / 5 deselected / 1 selected                                                                                                          

tests/foreman/ui/test_organization.py .     
================================================= 1 passed, 5 deselected in 271.93 seconds =================================================
```

### CODE REVIEW

File: tests/foreman/ui/test_organization.py [hunk 1]
```diff
@@ -35,12 +35,26 @@ def test_positive_end_to_end(session):
 
     :CaseLevel: Integration
 
-    :CaseImportance: High
+    :CaseImportance: Critical
```
Comment (omkarkhatavkar) [inline tests/foreman/ui/test_organization.py]: Merging everything will reduce the overall test execution time, but the single test execution time will increase, should we keep them in tier4 tests? (based on the time of execution)

File: tests/foreman/ui/test_organization.py [hunk 2]
```diff
@@ -35,12 +35,26 @@ def test_positive_end_to_end(session):
...
     """
     name = gen_string('alpha')
     new_name = gen_string('alpha')
     label = gen_string('alphanumeric')
     description = gen_string('alpha')
+
+    # entities to be added and removed
+    user = entities.User().create()
+    media = entities.Media(
+        path_=INSTALL_MEDIUM_URL % gen_string('alpha', 6),
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_organization.py]: can you change %-based formatting to .format?

File: tests/foreman/ui/test_organization.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/ui/test_organization.py]: Hi @omkarkhatavkar  I'd say it's still not so lengthy to qualify for tier4, and I'd like to get these tests executed sooner, given the gating logic we'd like to implement. If this causes delays in tier2 we can move it later.

Comment (omkarkhatavkar) [reply tests/foreman/ui/test_organization.py]: agreed! thanks for the update.

**Comment by pondrejk** [comment]
hi @JacobCallahan thanks for the review, I updated to use .format, would you mind revisiting?

**Comment by JacobCallahan** [comment]
Restarted failing Travis run and it is passing now. Merging.

**Pull request merged**